sdk/lib/crt/misc/dbgrpt.cpp

```diff
@@ -412,5 +418,68 @@ int __cdecl _CrtDbgReportW(int reportType, const wchar_t *filename, int linenumb
     return nResult;
 }
 
+EXTERN_C
+int __cdecl
+_CrtDbgReportV(
+    int reportType,
+    const char *filename,
+    int linenumber,
+    const char *moduleName,
+    const char *format,
+    va_list arglist)
+{
+    return _VCrtDbgReportA(reportType, filename, linenumber, moduleName, format, arglist);
```

Comment:
Why is this an extra function?

Reply:
> Also add the internal _VCrtDbgReportA and _VCrtDbgReportW functions listed in
https://learn.microsoft.com/en-us/cpp/c-runtime-library/internal-crt-globals-and-functions?view=msvc-170

from the 1st commit. But I'm fine to remove this indirection.

Reply:
ah, ok. No it's fine.